src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -113,28 +166,37 @@ ContinuousBatchingPipeline::ContinuousBatchingPipeline(
     auto draft_model_desr = extract_draft_model_from_config(properties_without_draft_model);
     auto is_prompt_lookup_enabled = extract_prompt_lookup_from_config(properties_without_draft_model);
     auto model = utils::singleton_core().read_model(model_str, weights_tensor);
+    auto directory = std::filesystem::path(get_directory(model_str));
```

Comment:
`model_str` is a content of XML file. So, we cannot extract `directory` here.

Since 2025.0 release, IR frontend inserts `__weights_path` as runtime info for ov::Model (see https://github.com/openvinotoolkit/openvino/pull/29101), so I think we can try to check this information and restore directory where model was read from

Reply:
Done.